tests/virt_autotest/sriov_network_card_pci_passthrough.pm

```diff
@@ -38,10 +38,17 @@ sub run_test {
     my $log_dir = "/tmp/sriov_pcipassthru";
     script_run "[ -d $log_dir ] && rm -rf $log_dir; mkdir -p $log_dir";
 
-    #get the SR-IOV device BDF and interface
-    my @host_pfs;
-    @host_pfs = find_sriov_ethernet_devices();
-
+    #check the SR-IOV device BDF and interface
+    my @host_pfs = find_sriov_ethernet_devices();
+    unless (@host_pfs) {
+        if (get_var("REGRESSION")) {
```

Comment:
Why not unify the handling for REGRESSION being true and false? Besides, please do not use fake poo#55555 for soft_fail.

Reply:
One thing should be noted that "die" will trigger post fail hook which collects and uploads logs and does other processing. In the case of  sriov support check,  it's unnecessary? It will takes 10+ minuets?  A "return" is a quick and clean finish. @Julie-CAO , would you still like "die" here?

Reply:
Yes, `die` is what we want to abort the test and mark the sriov test module fail. post_fail_hook might not be very necessary, but it might be helpful if the screenshots don't provide enough information. I am not quite sure about post_fail_hook, but a red test result is really required.

Reply:
Dropped the change.